src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -320,4 +323,30 @@ public static Set<Tag> parseTags(Collection<String> tags) throws ParseException
         }
         return tagSet;
     }
+
+    /**
+     * Parses {@code arg} into a {@code Order}.
```

Comment:
maybe can use (@String arg} ? Similar to the convention of java docs in ParserUtil